installing/installing_with_agent_based_installer/installing-with-agent-based-installer.adoc

```diff
@@ -6,13 +6,27 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+Use the following procedures to install an {product-title} cluster using the Agent-based Installer.
+
 [id="prerequisites_installing-with-agent-based-installer"]
 == Prerequisites
 
 * You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
 * You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
 
-include::modules/installing-ocp-agent.adoc[leveloffset=+1]
+// This anchor ID is extracted/replicated from the former installing-ocp-agent.adoc module to preserve links.
+[id="installing-ocp-agent_installing-with-agent-based-installer"]
+== Installing {product-title} with the Agent-based Installer
+
+The following procedures deploy a single-node {product-title} in a disconnected environment. You can use these procedures as a basis and modify according to your requirements.
+
+include::modules/installing-ocp-agent-download.adoc[leveloffset=+2]
+
+include::modules/installing-ocp-agent-boot.adoc[leveloffset=+2]
+
+include::modules/installing-ocp-agent-tui.adoc[leveloffset=+2]
+
+include::modules/installing-ocp-agent-verify.adoc[leveloffset=+2]
```

Comment:
It appears these were existing links, but [the first two](Additional resources) don't go to the specific module that the link text suggests. Is it worth fixing?